aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/IntegrationTestDependencies.kt

```diff
@@ -110,14 +110,14 @@ class S3TestDependencies(
     }
 }
 
-private val AsyncStd = CargoDependency("async-std", CratesIo("1.12"), scope = DependencyScope.Dev)
-private val AsyncStream = CargoDependency("async-stream", CratesIo("0.3"), DependencyScope.Dev)
-private val Criterion = CargoDependency("criterion", CratesIo("0.3"), scope = DependencyScope.Dev)
-private val FuturesCore = CargoDependency("futures-core", CratesIo("0.3"), DependencyScope.Dev)
-private val FuturesUtil = CargoDependency("futures-util", CratesIo("0.3"), scope = DependencyScope.Dev)
-private val Hound = CargoDependency("hound", CratesIo("3.4"), DependencyScope.Dev)
-private val SerdeJson = CargoDependency("serde_json", CratesIo("1"), features = emptySet(), scope = DependencyScope.Dev)
-private val Smol = CargoDependency("smol", CratesIo("1.2"), scope = DependencyScope.Dev)
-private val Tokio = CargoDependency("tokio", CratesIo("1"), features = setOf("macros", "test-util"), scope = DependencyScope.Dev)
-private val Tracing = CargoDependency("tracing", CratesIo("0.1"), scope = DependencyScope.Dev)
-private val TracingSubscriber = CargoDependency("tracing-subscriber", CratesIo("0.2"), scope = DependencyScope.Dev)
+private val AsyncStd = CargoDependency("async-std", CratesIo("1.12.0"), scope = DependencyScope.Dev)
+private val AsyncStream = CargoDependency("async-stream", CratesIo("0.3.0"), DependencyScope.Dev)
+private val Criterion = CargoDependency("criterion", CratesIo("0.3.0"), scope = DependencyScope.Dev)
+private val FuturesCore = CargoDependency("futures-core", CratesIo("0.3.0"), DependencyScope.Dev)
+private val FuturesUtil = CargoDependency("futures-util", CratesIo("0.3.0"), scope = DependencyScope.Dev)
+private val Hound = CargoDependency("hound", CratesIo("3.4.0"), DependencyScope.Dev)
+private val SerdeJson = CargoDependency("serde_json", CratesIo("1.0.0"), features = emptySet(), scope = DependencyScope.Dev)
+private val Smol = CargoDependency("smol", CratesIo("1.2.0"), scope = DependencyScope.Dev)
+private val Tokio = CargoDependency("tokio", CratesIo("1.0.0"), features = setOf("macros", "test-util"), scope = DependencyScope.Dev)
+private val Tracing = CargoDependency("tracing", CratesIo("0.1.0"), scope = DependencyScope.Dev)
+private val TracingSubscriber = CargoDependency("tracing-subscriber", CratesIo("0.3.5"), scope = DependencyScope.Dev, features = setOf("env-filter"))
```

Comment:
we should build a dependency version system

Reply:
Tracking in https://github.com/awslabs/smithy-rs/issues/1611